modules/planning/planner/em/em_planner.cc

```diff
@@ -138,6 +138,38 @@ void EMPlanner::RecordDebugInfo(ReferenceLineInfo* reference_line_info,
   ptr_stats->set_time_ms(time_diff_ms);
 }
 
+Status EMPlanner::Plan(const TrajectoryPoint& planning_start_point,
```

Comment:
I suggest make some name changes on this function if we move them to the same class. Two "Plan" functions are doing different things. Keeping same name would make developers hard to understand.